kano_apps/AppManage.py

```diff
@@ -118,15 +118,15 @@ def download_app(app_id_or_slug):
     icon_path = '/tmp/{}.{}'.format(app_id_or_slug, icon_file_type)
     rv, err = download_url(data['icon_url'], icon_path)
     if not rv:
-        msg = _("Unable to download the application ({})").format(err)
+        msg = _(u"Unable to download the application ({})").format(err)
```

Comment:
I've fallen a bit behind with this, so what's the convention generally? Pass `unicode` to `_()` or strings? Or both?